adafruit_button/button_base.py

```diff
@@ -78,7 +78,9 @@ def __init__(
     @property
     def label(self):
         """The text label of the button"""
-        return self._label.text
+        if self._label is not None and hasattr(self._label, "text"):
+            return self._label.text
+        return None
```

Comment:
could simply be `return getattr(self._label, "text", None)`, though this is fine too.

Reply:
Thank you! changed to getattr with the latest commit.